restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/proxy/ProxyManager.java

```diff
@@ -103,13 +104,18 @@ private Address contact(final Gateway gateway, final int expires) throws Servlet
         if (outboundInterface == null)
             outboundInterface = (SipURI) factory.createSipURI(null, address);
         final String user = gateway.getUserName();
+        String contactUser = new String (user);
```

Comment:
i think this "new String"  is not strictly neccesary...

Reply:
You are right, to be updated.